### PR TITLE
Ps 117 centreon ha 4nodes updates

### PR DESCRIPTION
**Pull request by cedricmeschin**
## Description

Updates on output commands
Ticket PS: https://centreon.atlassian.net/browse/PS-117

## Target version

- [ ] 21.10.x (staging)
- [ ] 22.04.x (staging)
- [X] 22.10.x (staging)
- [ ] Cloud (staging)
- [ ] Plugin Packs (staging)
- [ ] 23.04.x (next)

### CODE REVIEW

File: versioned_docs/version-22.10/installation/installation-of-centreon-ha/installation-2-nodes.md [hunk 1]
```diff
@@ -1529,9 +1530,9 @@ Ticket Constraints:
 ```bash
 Location Constraints:
 Ordering Constraints:
-  pcs constraint order stop centreon then demote ms_mysql-master
+  stop centreon then demote ms_mysql-master
 Colocation Constraints:
-  ms_mysql-master with centreon (score:INFINITY) (rsc-role:Master) (with-rsc-role:Started)
+  ms_mysql-clone with centreon (score:INFINITY) (rsc-role:Master) (with-rsc-role:Started)
```
Comment (jeanrenemoyap) [inline versioned_docs/version-22.10/installation/installation-of-centreon-ha/installation-2-nodes.md:1535]: ms_mysql-**master** with centreon (score:INFINITY) (rsc-role:Master) (with-rsc-role:Started)

**Review by jeanrenemoyap — CHANGES_REQUESTED** [review]

File: versioned_docs/version-22.10/installation/installation-of-centreon-ha/installation-2-nodes.md (hunk 1 above)
Comment (cedricmeschin) [reply versioned_docs/version-22.10/installation/installation-of-centreon-ha/installation-2-nodes.md:1535]: It's ms_mysql-clone on RHEL8 but ms_mysql-master on CentOS7

File: i18n/fr/docusaurus-plugin-content-docs/version-22.10/installation/installation-of-centreon-ha/installation-4-nodes.md [hunk 2]
```diff
@@ -1769,12 +1771,6 @@ Ces actions uniquement sur le `@CENTRAL_MASTER_NAME@` puis les fichiers de confi
...
 
-Redémarrer ensuite les services Centreon à l'aide de la commande :
-
-```bash
-pcs resource restart centreon
-```
-
 ### Modification des 3 fichiers de configuration
 
 Après avoir modifié le output du broker, il nous faut modifier les fichiers de configuration de Centreon.
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs/version-22.10/installation/installation-of-centreon-ha/installation-4-nodes.md:1776]:
```suggestion
Après avoir modifié l'output du broker, il nous faut modifier les fichiers de configuration de Centreon.
```

File: i18n/fr/docusaurus-plugin-content-docs/version-22.10/installation/installation-of-centreon-ha/installation-4-nodes.md [hunk 3]
```diff
@@ -51,6 +51,8 @@ Pour le bon fonctionnement du cluster, en particulier pour éviter les cas de sp
 Afin de respecter les meilleures pratiques et de bénéficier d'une infrastructure aussi résiliente que possible, le placement du serveur
 Quorum doit être sur un site différent des deux nœuds principaux, avec des attachements réseaux indépendants.
 
+> **AVERTISSEMENT:** Assurez-vous que Selinux et Firewalld sont désactivés.
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs/version-22.10/installation/installation-of-centreon-ha/installation-4-nodes.md:54]:
```suggestion
> **AVERTISSEMENT:** Assurez-vous que SELinux et Firewalld sont désactivés.
```

File: i18n/fr/docusaurus-plugin-content-docs/version-22.10/installation/installation-of-centreon-ha/installation-4-nodes.md [hunk 4]
```diff
@@ -1856,6 +1852,60 @@ database:
...
 ```
 
+Ensuite, vous devez redémarrer les ressources gorgone et cbd_central_broker pour que les changements prennent effet.
+Le redémarrage de la ressource gorgone redémarre toutes les ressources suivantes, donc en utilisant la commande suivante :
+
+```bash
+pcs resource restart gorgone
+```
+
+Et après le redémarrage des ressources, vérifiez si tout est OK avec la commande `crm_mon -fr`, le résultat devrait être comme ceci :
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs/version-22.10/installation/installation-of-centreon-ha/installation-4-nodes.md:1862]:
```suggestion
Et après le redémarrage des ressources, vérifiez si tout est OK avec la commande `crm_mon -fr`, le résultat devrait être similaire à ceci :
```

File: i18n/fr/docusaurus-plugin-content-docs/version-22.10/installation/installation-of-centreon-ha/installation-4-nodes.md [hunk 5]
```diff
@@ -1856,6 +1852,60 @@ database:
     password: "@MARIADB_CENTREON_PASSWD@"
 ```
 
+Ensuite, vous devez redémarrer les ressources gorgone et cbd_central_broker pour que les changements prennent effet.
+Le redémarrage de la ressource gorgone redémarre toutes les ressources suivantes, donc en utilisant la commande suivante :
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs/version-22.10/installation/installation-of-centreon-ha/installation-4-nodes.md:1856]:
```suggestion
Utilisez la commande suivante pour redémarrer la ressource gorgone et toutes les ressources suivantes :
```

File: versioned_docs/version-22.10/installation/installation-of-centreon-ha/installation-4-nodes.md [hunk 6]
```diff
@@ -51,6 +51,8 @@ In order to keep the cluster safe from split-brain issues, a third server is man
 In order to adhere to best practices and be as resilient as possible, the Quorum server placement
 Quorum server should be at a different site than the two primary nodes, with independent network attachments.
 
+> **WARNING:** Be sure Selinux and Firewalld are disabled.
```
Comment (cg-tw) [inline versioned_docs/version-22.10/installation/installation-of-centreon-ha/installation-4-nodes.md:54]:
```suggestion
> **WARNING:** Be sure SELinux and Firewalld are disabled.
```

File: versioned_docs/version-22.10/installation/installation-of-centreon-ha/installation-4-nodes.md [hunk 7]
```diff
@@ -1851,6 +1847,60 @@ database:
     password: "@MARIADB_CENTREON_PASSWD@"
 ```
 
+And then you need to restart gorgone and cbd_central_broker resources to changes take effects.
```
Comment (cg-tw) [inline versioned_docs/version-22.10/installation/installation-of-centreon-ha/installation-4-nodes.md:1850]:
```suggestion
And then you need to restart the gorgone and cbd_central_broker resources for changes to take effect.
```

File: versioned_docs/version-22.10/installation/installation-of-centreon-ha/installation-4-nodes.md [hunk 8]
```diff
@@ -1851,6 +1847,60 @@ database:
...
+Restarting gorgone resource will restart all resources below, so using the following command:
```
Comment (cg-tw) [inline versioned_docs/version-22.10/installation/installation-of-centreon-ha/installation-4-nodes.md:1851]:
```suggestion
Use the following command to restart the gorgone resource and all subsequent resources:
```

File: versioned_docs/version-22.10/installation/installation-of-centreon-ha/installation-4-nodes.md [hunk 9]
```diff
@@ -1851,6 +1847,60 @@ database:
...
+
+```bash
+pcs resource restart gorgone
+```
+
+And after resources are restarted, verify if all is OK with the command `crm_mon -fr`, the result should be like that:
```
Comment (cg-tw) [inline versioned_docs/version-22.10/installation/installation-of-centreon-ha/installation-4-nodes.md:1857]:
```suggestion
After resources are restarted, verify if all is OK using the command `crm_mon -fr`. The result should look like this:
```

File: versioned_docs/version-22.10/installation/installation-of-centreon-ha/installation-4-nodes.md [hunk 10]
```diff
@@ -1851,6 +1847,60 @@ database:
...
+    * centreon_central_sync     (systemd:centreon-central-sync):         Started @CENTRAL_MASTER_NAME@
+    * cbd_central_broker        (systemd:cbd-sql):       Started @CENTRAL_MASTER_NAME@
+    * centengine        (systemd:centengine):    Started @CENTRAL_MASTER_NAME@
+    * centreontrapd     (systemd:centreontrapd):         Started@CENTRAL_MASTER_NAME@
+    * snmptrapd (systemd:snmptrapd):     Started @CENTRAL_MASTER_NAME@
+
+Migration Summary:
+```
+
+If an error is found while your resources are running, try "cleaning" with this command:
```
Comment (cg-tw) [inline versioned_docs/version-22.10/installation/installation-of-centreon-ha/installation-4-nodes.md:1898]:
```suggestion
If an error is found while your resources are running, try doing some "cleaning" using this command:
```

**Review by cg-tw — CHANGES_REQUESTED** [review]

**Review by cg-tw — APPROVED** [review]

**Review by psamecentreon — APPROVED** [review]

**Review by cg-tw — APPROVED** [review]

**Pull request merged**